QKSMS/src/main/java/com/moez/QKSMS/common/formatter/FormatterFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.moez.QKSMS.common.formatter;
+
+public class FormatterFactory {
+    private static final Formatter[] FORMATTERS = {new Number2ContactFormatter()};
+
+    public static String format(String text) {
+        for (Formatter formatter : FORMATTERS)
```

Comment:
Can you add braces around any blocks that have code on separate lines?

For example:

Good
```
for (Formatter formatter : FORMATTERS) {
    text = formatter.format(text);
}
```

Okay
```
for (Formatter formatter : FORMATTERS) text = formatter.format(text);
```

Not so good
```
for (Formatter formatter : FORMATTERS)
    text = formatter.format(text);
```